update-magni-image.sh

```diff
@@ -31,11 +31,8 @@ echo "Installing extra packages"
 sudo apt-get install ros-kinetic-teleop-twist-keyboard
 
 # If current packages are not available via debs, then build from source
-if [[ ! `apt-cache policy ros-kinetic-move-basic` =~ "Installed: 0.2.2" ]]; then
-   get_source 'move_basic'
-fi
-if [[ ! `apt-cache policy ros-kinetic-magni-robot` =~ "Installed: 0.2.1" ]]; then
-   get_source 'magni_robot'
+if [[ ! `apt-cache policy ros-kinetic-ubiquity-motor` =~ "Installed: 0.6.1" ]]; then
+   get_source 'ubiquity-motor'
```

Comment:
Should be ubiquity_motor

Reply:
I agree that giving people source with no easy way to get back to debs is not a good solution. If you can put a deb on packages.ubiquityrobotics.com until the OSRF sync happens, that is much cleaner.